bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -1931,4 +1932,17 @@ private function addGotenbergNode(ArrayNodeDefinition $rootNode): void
             ->end()
         ->end();
     }
+    private function addChromiumNode(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+            ->arrayNode('chromium')
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->scalarNode('uri')
+                ->defaultNull()
+                ->end()
+            ->end()
+        ->end();
```

Comment:
Use correct indentation
```suggestion
            ->children()
                ->arrayNode('chromium')
                    ->addDefaultsIfNotSet()
                    ->children()
                        ->scalarNode('uri')
                        ->defaultNull()
                    ->end()
                ->end()
            ->end();
```

Reply:
I think it is also a `end()` missing, right?
```suggestion
            ->children()
                ->arrayNode('chromium')
                    ->addDefaultsIfNotSet()
                    ->children()
                        ->scalarNode('uri')
                            ->defaultNull()
                        ->end()
                    ->end()
                ->end()
            ->end();
```

Reply:
Also for addGotenbergNode